### PR TITLE
Copy top menu from idr.openmicroscopy.org

### PR DESCRIPTION
**Pull request by manics**
Also makes deployment a dropdown (IDR, VAE)

### CODE REVIEW

File: src/_includes/head-idr.html [hunk 1]
```diff
@@ -35,7 +35,26 @@
...
+                        <ul class="submenu menu vertical" data-submenu>
+                            <li><a href="https://idr.openmicroscopy.org/about/api.html">API Access</a></li>
+                            <li><a href="https://idr.openmicroscopy.org/about/itr.html">Image Tools Resource (ITR)</a></li>
+                            <li><a href="/">Virtual Analysis Environment (VAE)</a></li>
+                        </ul>
+                    </li>
+                    <li class="has-submenu"><a>Deployment</a>
+                        <ul class="submenu menu vertical" data-submenu>
+                            <li><a href="https://idr.openmicroscopy.org/about/deployment.html">IDR</a></li>
+                            <li><a href="deployment.html">VAE</a></li>
```
Comment (mtbc) [inline src/_includes/head-idr.html]: expand acronym here too?

**Comment by manics** [comment]
This is the final proposed website for IDR-0.5.0

File: src/_includes/head-idr.html (hunk 1 above)
Comment (manics) [reply src/_includes/head-idr.html]: Expand IDR too? Or just VAE?

Comment (mtbc) [reply src/_includes/head-idr.html]: I think IDR too. Helps us to remember if it's a repository or a resource. :smiley:

Comment (manics) [reply src/_includes/head-idr.html]: Done

File: src/_includes/head-idr.html [hunk 2]
```diff
@@ -35,7 +35,26 @@
...
+                            <li><a href="https://idr.openmicroscopy.org/about/submission.html">Overview</a></li>
+                            <li><a href="https://idr.openmicroscopy.org/about/screens.html">Screens</a></li>
+                            <li><a href="https://idr.openmicroscopy.org/about/experiments.html">Experiments</a></li>
+                        </ul>
+                    </li>
+                    <li class="has-submenu"><a>Analysis</a>
+                        <ul class="submenu menu vertical" data-submenu>
+                            <li><a href="https://idr.openmicroscopy.org/about/api.html">API Access</a></li>
+                            <li><a href="https://idr.openmicroscopy.org/about/itr.html">Image Tools Resource (ITR)</a></li>
+                            <li><a href="/">Virtual Analysis Environment (VAE)</a></li>
```
Comment (mtbc) [inline src/_includes/head-idr.html:49]: A link to `/`?

Comment (manics) [reply src/_includes/head-idr.html:49]: Yes, it's to keep the menus the same as https://idr.openmicroscopy.org/about/ (though the change to the deployment menu obviously breaks this)

Comment (manics) [reply src/_includes/head-idr.html:49]: Current site is https://idr-analysis.openmicroscopy.org/

Comment (mtbc) [reply src/_includes/head-idr.html:49]: Ah, that makes sense, thank you.

**Review by joshmoore — APPROVED** [review]
Deployed locally in docker; looks good. Only issue at the moment is that the idr.o.org `Deployment` menu now differs. Likely something we will want to address (by merging the repos, by refactoring out the common code, etc)

**Comment by joshmoore** [comment]
screen shot 2018-05-09 at 13 08 34

**Pull request merged**